Test failure JIT/opt/Multiply/IntMultiply/IntMultiply.sh

Run: runtime-coreclr jitstress 20221019.2

Failed test:
```
coreclr Linux x64 Checked jitminopts @ Ubuntu.1804.Amd64.Open

- JIT/opt/Multiply/IntMultiply/IntMultiply.sh

coreclr windows x64 Checked jitminopts @ Windows.10.Amd64.Open

- JIT\\opt\\Multiply\\IntMultiply\\IntMultiply.cmd

coreclr OSX x64 Checked jitminopts @ OSX.1200.Amd64.Open

- JIT/opt/Multiply/IntMultiply/IntMultiply.sh
```
**Error message:**
```
__tmp1_IntMultiply.cs:23:15: error: X64-NEXT: expected string not found in input
// X64-NEXT: shl [[REG0]], 32
^
__jit_disasm.out:53:2: note: scanning from here
imul rax, qword ptr [rbp-08H]
^
__jit_disasm.out:53:2: note: with "REG0" equal to "eax"
imul rax, qword ptr [rbp-08H]
^
__jit_disasm.out:84:2: note: possible intended match here
shl rax, 32
^

Input file: __jit_disasm.out
Check file: __tmp1_IntMultiply.cs

-dump-input=help explains the following input dump.

Input was:
<<<<<<
.
.
.
13: G_M20640_IG01:
14:  push rbp
15:  sub rsp, 16
16:  lea rbp, [rsp+10H]
17:  mov dword ptr [rbp-04H], edi
18:  ;; size=13 bbWeight=1 PerfScore 2.75
19: G_M20640_IG02:
20:  mov eax, dword ptr [rbp-04H]
21:  neg eax
22:  ;; size=5 bbWeight=1 PerfScore 1.25
23: G_M20640_IG03:
24:  add rsp, 16
25:  pop rbp
26:  ret
27:  ;; size=6 bbWeight=1 PerfScore 1.75
28:
29: ; Total bytes of code 24, prolog size 10, PerfScore 8.15, instruction count 9, allocated bytes for code 24 (MethodHash=7b03af5f) for method CodeGenTests.IntMultiply:UInt32_MultiplyWithUInt32MaxValue(uint):uint
30: ; ============================================================
31:
32: ; Assembly listing for method CodeGenTests.IntMultiply:UInt64_MultiplyWithUInt32MaxValue(ulong):ulong
33: ; Emitting BLENDED_CODE for X64 CPU with AVX - Unix
34: ; MinOpts code
35: ; rbp based frame
36: ; partially interruptible
37: ; Final local variable assignments
38: ;
39: ; V00 arg0 [V00 ] ( 1, 1 ) long -> [rbp-08H] do-not-enreg[]
40: ;# V01 OutArgs [V01 ] ( 1, 1 ) lclBlk ( 0) [rsp+00H] do-not-enreg[] "OutgoingArgSpace"
41: ;
42: ; Lcl frame size = 16
43:
44: G_M35971_IG01:
45:  push rbp
46:  sub rsp, 16
47:  lea rbp, [rsp+10H]
48:  mov qword ptr [rbp-08H], rdi
49:  ;; size=14 bbWeight=1 PerfScore 2.75
50: G_M35971_IG02:
51:  mov eax, -1
52:  mov eax, eax
53:  imul rax, qword ptr [rbp-08H]
next:23'0      X~~~~~~~~~~~~~~~~~~~~~~~~~~~~ error: no match found
next:23'1                                    with "REG0" equal to "eax"
54:  ;; size=12 bbWeight=1 PerfScore 4.50
next:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
55: G_M35971_IG03:
next:23'0     ~~~~~~~~~~~~~~
56:  add rsp, 16
next:23'0     ~~~~~~~~~~~~
57:  pop rbp
next:23'0     ~~~~~~~~
58:  ret
next:23'0     ~~~~~
59:  ;; size=6 bbWeight=1 PerfScore 1.75
next:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
60:
next:23'0     ~
61: ; Total bytes of code 32, prolog size 10, PerfScore 12.20, instruction count 10, allocated bytes for code 32 (MethodHash=175b737c) for method CodeGenTests.IntMultiply:UInt64_MultiplyWithUInt32MaxValue(ulong):ulong
next:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
62: ; ============================================================
next:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
63:
next:23'0     ~
64: ; Assembly listing for method CodeGenTests.IntMultiply:UInt64_MultiplyWithUInt32MaxValuePlusOne(ulong):ulong
next:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
65: ; Emitting BLENDED_CODE for X64 CPU with AVX - Unix
next:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
66: ; MinOpts code
next:23'0     ~~~~~~~~~~~~~~
67: ; rbp based frame
n


Stack trace
   at JIT_opt._Multiply_IntMultiply_IntMultiply_._Multiply_IntMultiply_IntMultiply_sh()
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

Queued | OS | Arch | Column1
-- | -- | -- | --
2022-10-20T10:30:32.649Z | ubuntu.1804.amd64.open.rt | x64 | [runtime-coreclr pgo   Checked-nopgo](https://dev.azure.com/dnceng/public/_build/results?buildId=58043&view=ms.vss-test-web.build-test-results-tab&runId=1191774&resultId=110090&paneView=debug)
2022-10-20T06:27:21.942Z | osx.1200.amd64.open | x64 | [runtime-coreclr jitstress   Checked-jitminopts](https://dev.azure.com/dnceng/public/_build/results?buildId=57962&view=ms.vss-test-web.build-test-results-tab&runId=1188938&resultId=109439&paneView=debug)
2022-10-20T05:58:36.71Z | ubuntu.1804.amd64.open.rt | x64 | [runtime-coreclr jitstress   Checked-jitminopts](https://dev.azure.com/dnceng/public/_build/results?buildId=57962&view=ms.vss-test-web.build-test-results-tab&runId=1188792&resultId=105604&paneView=debug)

--- a/src/tests/JIT/opt/Multiply/IntMultiply.csproj
+++ b/src/tests/JIT/opt/Multiply/IntMultiply.csproj
@@ -11,7 +11,7 @@
       <HasDisasmCheck>true</HasDisasmCheck>
     </Compile>
 
-    <CLRTestEnvironmentVariable Include="COMPlus_TieredCompilation" Value="0" />
-    <CLRTestEnvironmentVariable Include="COMPlus_JITMinOpts" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_JITMinOpts" Value="0" />
   </ItemGroup>
 </Project>

